Library/JsonConfig.ts

```diff
@@ -130,6 +134,12 @@ export class JsonConfig implements IJsonConfig {
             if (jsonConfig.noPatchModules != undefined) {
                 this.noPatchModules = jsonConfig.noPatchModules;
             }
+            if (jsonConfig.enableAutoWebSnippetInjection != undefined) {
+                this.enableAutoWebSnippetInjection = jsonConfig.enableAutoWebSnippetInjection;
+            }
+            if (jsonConfig.isDebugWebSnippet !== undefined) {
```

Comment:
I will expect enableAutoWebSnippetInjection to control this one as well, it is weird user add env variable to turn the feature off and debug snippet is still added, add extra check to avoid this